### PR TITLE
chore: add types to ipx provider options

### PR DESCRIPTION
**Pull request by aaharu**
rework of https://github.com/nuxt/image/pull/976

> https://image.nuxt.com/get-started/configuration#modifiers
> According to the document, setting to default modifiers should be possible, but it wasn't working in typescript, so I fixed it.

### CODE REVIEW

**Review by Atinux — APPROVED** [review]

File: src/types/module.ts [hunk 1]
```diff
@@ -65,6 +65,39 @@ export interface UploadcareOptions {
...
+  negate: true | 'true'
+  normalize: true | 'true'
+  threshold: number | string
+  modulate: string
+  tint: number | string
+  grayscale: true | 'true'
+  animated: true | 'true'
+}
+
+export interface StaticIPXOptions extends Omit<IPXRuntimeConfig, 'alias'> {
```
Comment (pi0) [inline src/types/module.ts:97]: Since after refactors i guess simply call interface IPXOtions..

File: src/types/module.ts [hunk 2]
```diff
@@ -65,6 +65,39 @@ export interface UploadcareOptions {
   [key: string]: any
 }
 
+export interface IPXModifiers extends ImageModifiers {
```
Comment (pi0) [inline src/types/module.ts:68]: Would you please add a comment above to ipx docs and if had time open an issue upstream to remember adding operation types from upstream? 🙏

File: src/types/module.ts [hunk 3]
```diff
@@ -85,8 +118,8 @@ export interface ImageProviders {
   strapi?: any,
   imageengine?: any,
   uploadcare?: Partial<UploadcareOptions>,
-  ipx?: Partial<IPXRuntimeConfig>
-  static?: Partial<IPXRuntimeConfig>
+  ipx?: Partial<StaticIPXOptions>
+  static?: Partial<StaticIPXOptions>
```
Comment (danielroe) [inline src/types/module.ts:122]:
```suggestion
  ipx?: Partial<IPXOptions>
  static?: Partial<IPXOptions>
```

File: src/types/module.ts (hunk 1 above)
Comment (danielroe) [inline src/types/module.ts:97]:
```suggestion
export interface IPXOptions extends Omit<IPXRuntimeConfig, 'alias'> {
```

File: src/types/module.ts (hunk 2 above)
Comment (pi0) [inline src/types/module.ts:68]:
```suggestion
// Reference: https://github.com/unjs/ipx?tab=readme-ov-file#modifiers
// TODO: https://github.com/unjs/ipx/issues/199
export interface IPXModifiers extends ImageModifiers {
```

**Review by pi0 — APPROVED** [review]

**Pull request merged**